src/components/AvatarPicker/AvatarPicker.css

```diff
@@ -1,9 +1,9 @@
-.avatar-picker__avatar-list {
+.avatar-picker {
   display: flex;
   justify-content: center;
 }
 
-.avatar-picker__avatar-select {
+.avatar-picker__avatar-option {
```

Comment:
nit: I still think this should just be `.avatar-picker__option`

I think `.avatar-picker__avatar-option` is redundant. You already imply that the option will be an avatar?